builtin/logical/ssh/backend_test.go

```diff
@@ -1893,6 +2034,66 @@ func getSshCaTestCluster(t *testing.T, userIdentity string) (*vault.TestCluster,
 	return cluster, userpassToken
 }
 
+// TKTK
```

Comment:
Could this comment be removed please?

Reply:
Sorry about that! I've removed it in 6ecf401852